controllers/platform/k8s.go

```diff
@@ -233,8 +233,12 @@ func createOrUpdateService(ctx context.Context, client client.Client, platform *
 
 func getLabels(platform *operatorapi.SonataFlowPlatform, psh services.PlatformServiceHandler) (map[string]string, map[string]string) {
 	lbl := map[string]string{
-		workflowproj.LabelApp:     platform.Name,
-		workflowproj.LabelService: psh.GetServiceName(),
+		workflowproj.LabelApp:          platform.Name,
```

Comment:
I think we can use the new k8s label in place of this. Can you remove it?

Reply:
I think there are varioius selector on "app"
https://github.com/search?q=repo%3Aapache%2Fincubator-kie-kogito-serverless-operator+%5C%22app%5C%22&type=code

Reply:
if this look okay I'll remove it